packages/cli/index.js

```diff
@@ -155,9 +160,14 @@ program
 
     const changeHandler = (filePath) => {
       logger.info(`[${new Date().toLocaleTimeString()}] Reload for file change: ${filePath}`);
-      logger.info('Synchronizing opened pages list before reload');
-      const normalizedActiveUrls = liveServer.getActiveUrls().map(url => fsUtil.removeExtension(url));
-      site.changeCurrentOpenedPages(normalizedActiveUrls);
+      if (onePagePath) {
+        logger.info('Synchronizing opened pages list before reload');
+        const normalizedActiveUrls = liveServer.getActiveUrls().map((url) => {
+          const completeUrl = path.extname(url) === '' ? path.join(url, 'index') : url;
```

Comment:
let's separate these unrelated ones to another PR with another appropriate commit message though

Reply:
I think that's better. I will do a revert commit and put this as a new minor PR.